content/en/blog/extend-falco-inputs-with-a-plugin-the-basics.md

```diff
@@ -5,7 +5,9 @@ Author: Thomas Labarussias
 slug: extend-falco-inputs-with-a-plugin-the-basics
 ---
 
-> This post is the first of a series about `How to develop Falco plugins`. It's adressed to anybody who would like to understand how plugins are written and want to contribute.
+> This post is is part of a series of articles about `How to develop Falco plugins`. It's adressed to anybody who would like to understand how plugins are written and want to contribute.
+> See other articles:
+> * [Extend Falco inputs by creating a Plugin: Register the plugin]({{< ref "/blog/extend-falco-inputs-with-a-plugin-register" >}})
```

Comment:
Would better to also add this reference by creating a new `Furthermore`/`Next Steps` section in the end of the article.